fagiServer/src/test/java/com/fagi/handler/inputhandler/UserNameAvailableRequestTest.java

```diff
@@ -0,0 +1,65 @@
+package com.fagi.handler.inputhandler;
+
+import com.fagi.handler.ConversationHandler;
+import com.fagi.handler.InputHandler;
+import com.fagi.model.Data;
+import com.fagi.model.User;
+import com.fagi.model.UserNameAvailableRequest;
+import com.fagi.responses.AllIsWell;
+import com.fagi.responses.UserExists;
+import com.fagi.util.OutputAgentTestUtil;
+import com.fagi.worker.InputAgent;
+import com.fagi.worker.OutputAgent;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.when;
+
+public class UserNameAvailableRequestTest {
+    private OutputAgent outputAgent;
+    private Data data;
+    private InputHandler inputHandler;
+
+    @BeforeEach
+    void setup() {
+        data = Mockito.mock(Data.class);
+        InputAgent inputAgent = Mockito.mock(InputAgent.class);
```

Comment:
Det kunne godt ligne at vi kunne have glæde af en BaseTestClass for InputHandler. Vi den samme kode i setup i samtlige klasser

Reply:
Ja det var nok en god ide